.github/workflows/pytest.yml

```diff
@@ -29,7 +29,7 @@ jobs:
     - name: Run tests
       run: |
         if [ ${{ matrix.code-cov }} ]; then codecov='--cov=autoPyTorch --cov-report=xml'; fi
-        python -m pytest --durations=20 --timeout=300 --timeout-method=thread -v $codecov test
+        python -m pytest --durations=20 --timeout=600 --timeout-method=signal -v $codecov test
```

Comment:
just a general question, why are we moving to timeout method signal?

Reply:
got the answer while reading the group chat. Thanks :P

Reply:
According to the documentation [here](https://pypi.org/project/pytest-timeout/):

> The benefit of this method is that the pytest process is not terminated and the test run can complete normally.

So it sounds like the test will still fail on a timeout, BUT all other test will complete. So this is great to see if everything is working except a particular test. Today, without this, pytest just dies.

Reply:
ohh okay. Thanks for the explanation.